meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -31,6 +31,9 @@ SDKMACHINE ?= "x86_64"
 # Generate source mirror tarballs:
 BB_GENERATE_MIRROR_TARBALLS = "1"
 
+# Do not allow building of anything that hasn't been checked beforehand.
+SUPPORTED_RECIPES_CHECK = "fatal"
+
 # Configure images in "development mode".
```

Comment:
Even enabling something that then most users will want to disable because it gets in their way would not be good user experience.

The way the mechanism works at the moment is such that it does not influence SSTATE, so while I understand the concern in principle, it does not apply here. It also does not influence the choice which version of a recipe gets built - I'll check that, but I suspect that we'll need to use blacklisting or layer priorities/ordering if we still need that kind of control.

Yes, multiple file lists are supported.